app/models/document_type/organisations_field.rb

```diff
@@ -16,4 +16,8 @@ def document_type
   def pre_update_issues(_edition, _params)
     Requirements::CheckerIssues.new
   end
+
+  def pre_publish_issues(edition)
+    pre_update_issues(edition, edition.tags.symbolize_keys)
```

Comment:
Minor: this seems a little premature (and a little complex), given that we don't have a definition of pre-publish issues for most of these fields; it may be that they differ, like they do for [the body field](https://github.com/alphagov/content-publisher/blob/master/app/models/document_type/body_field.rb#L32). Can we explicitly return no issues to indicate they have none defined?